### PR TITLE
Add Input Component

### PR DESCRIPTION
**Pull request by aesha-spreadd**
Add Input Component

Screenshot 2023-12-28 at 2 14 56 PM

Screenshot 2024-01-02 at 2 31 43 PM

- [x]  Checking the code is duplicate or not?
- [x]  Is these code has bug
- [ ]  Is code is check by shubhum sir or ankit sir or tara ma'am?
- [ ]  Are you going to merge this or not?

### CODE REVIEW

File: src/Input/Input.Styled.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components';
+import theme from '../theme';
+
+export const StyledInputComponent = styled.div`
+  column-gap: 10px;
+  flex-wrap: wrap;
+  font-family: 'Rubik', sans-serif;
+  font-size: 15px;
+  height: 182px;
```
Comment (ankitadrixus08) [inline src/Input/Input.Styled.js]: may be don't need height here @aeshapatel025

File: src/Input/Input.Styled.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  justify-content: center;
+  padding: 15px 50px;
+  row-gap: 10px;
+  width: 800px;
```
Comment (ankitadrixus08) [inline src/Input/Input.Styled.js]: use max width for this with width:100%; @aeshapatel025

File: src/Input/Input.Styled.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+export const StyledInputComponent = styled.div`
+  column-gap: 10px;
+  flex-wrap: wrap;
+  font-family: 'Rubik', sans-serif;
+  font-size: 15px;
+  height: 182px;
+  justify-content: center;
+  padding: 15px 50px;
+  row-gap: 10px;
```
Comment (ankitadrixus08) [inline src/Input/Input.Styled.js]: why use ro-gap, column-gap here @aeshapatel025

File: src/Input/Input.Styled.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    margin: 0 0 8px 0;
+    font-size: 15px;
+    color: ${(props) => (props.color ? theme.primary : 'black')};
+  }
+`;
+
+export const Input_Container_inputs_type = styled.div`
+  flex-direction: column;
+  flex-wrap: nowrap;
+  height: 77px;
```
Comment (ankitadrixus08) [inline src/Input/Input.Styled.js]: same as above @aeshapatel025

File: src/Input/Input.stories.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+export default {
+  title: 'Input',
+  component: Input,
+};
+
+export const Inputs = () => (
+  <div>
+    <Input_Container_inputs_type>
+      <Input label="Username*" />
```
Comment (ankitadrixus08) [inline src/Input/Input.stories.js]: use name, placeholder here @aeshapatel025

File: src/Input/Input.js [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { StyledInputComponent, StyledSpan } from './Input.Styled';
+
+const Input = ({ label, color, spantext }) => {
+  return (
+    <StyledInputComponent color={color}>
+      <label>{label}</label>
+      <input type="text" />
```
Comment (ankitadrixus08) [inline src/Input/Input.js]: need name and placeholder here @aeshapatel025

Comment (aesha-spreadd) [reply src/Input/Input.js]: changes done!

File: src/Input/Input.stories.js (hunk 5 above)
Comment (aesha-spreadd) [reply src/Input/Input.stories.js]: changes done!

File: src/Input/Input.Styled.js (hunk 4 above)
Comment (aesha-spreadd) [reply src/Input/Input.Styled.js]: changes done

File: src/Input/Input.Styled.js (hunk 1 above)
Comment (aesha-spreadd) [reply src/Input/Input.Styled.js]: yes ! i have removed it!

File: src/Input/Input.Styled.js (hunk 2 above)
Comment (aesha-spreadd) [reply src/Input/Input.Styled.js]: changes done

File: src/Input/Input.Styled.js (hunk 3 above)
Comment (aesha-spreadd) [reply src/Input/Input.Styled.js]: to give gap but i do not need it so i removed it

**Review by ankitadrixus08 — APPROVED** [review]

File: src/Input/Input.Styled.js [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+  label {
+    display: block;
+    margin: 0 0 8px 0;
+    font-size: 15px;
+    color: ${(props) => (props.color ? theme.primary : 'black')};
+  }
+`;
+
+export const Input_Container_inputs_type = styled.div`
```
Comment (tarakhant1297) [inline src/Input/Input.Styled.js]: @aeshapatel025 give proper name to this styled component

File: src/Input/Input.js [hunk 8]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { StyledInputComponent, StyledSpan } from './Input.Styled';
+
+const Input = ({ label, color, spantext, name, placeholder }) => {
+  return (
+    <StyledInputComponent color={color} name={name}>
+      <label>{label}</label>
```
Comment (tarakhant1297) [inline src/Input/Input.js]: @aeshapatel025 we don't put label in input component

File: src/Input/Input.stories.js [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+  <div>
+    <Input_Container_inputs_type>
+      <Input label="Username*" name="username" placeholder="Enter name here" />
+    </Input_Container_inputs_type>
+    <Input_Container_inputs_type>
+      <Input
+        label="Password*"
+        name="password"
+        placeholder="Enter password here"
+        color
```
Comment (tarakhant1297) [inline src/Input/Input.stories.js]: @aeshapatel025 why you add this

**Review by tarakhant1297 — COMMENTED** [review]
@aeshapatel025 https://ant.design/components/input follow this

File: src/Input/Input.stories.js (hunk 9 above)
Comment (aesha-spreadd) [reply src/Input/Input.stories.js]: ankit sir told me to add

**Review by tarakhant1297 — CHANGES_REQUESTED** [review]
https://ant.design/components/input not follow pros as antd input how i add prefix and suffix icon input if i want to add

File: src/Input/Input.js (hunk 8 above)
Comment (aesha-spreadd) [reply src/Input/Input.js]: changes done

**Review by tarakhant1297 — APPROVED** [review]

**Pull request merged**